src/App.css

```diff
@@ -0,0 +1,19 @@
+body {
+  background: rgb(236, 236, 236);
+}
+.App {
+  text-align: center;
+}
+
```

Comment:
These number are a bit crazy? Can you not achieve the same thing using `flexbox`?